docs/website/docs/plus/getting-started/tutorial.md

```diff
@@ -0,0 +1,385 @@
+
+```sh
+.
+├── .dlt/                 # your dlt settings including profile settings
+│   ├── config.toml
+│   ├── dev.secrets.toml
+│   └── secrets.toml
+├── _storage/             # local storage for your project, excluded from git
+├── destinations/         # your destinations, empty in this example
+├── sources/              # your sources, contains the code for the arrow source
+│   └── arrow.py
+├── .gitignore
+└── dlt.yml       # the main project manifest
+```
+
+### Understanding `dlt.yml`
+The `dlt.yml` file is the central configuration for your dlt+ Project. It defines the pipelines, sources, and destinations. In the generated project, the file looks like this:
+
+```yaml
+# project settings
+project:
+  name: template
+
+profiles:
+  # profiles allow you to configure different settings for different environments
+  dev: {}
+
+# your sources are the data sources you want to load from
+sources:
+  arrow:
+    type: sources.arrow.source
+
+# your destinations are the databases where your data will be saved
+destinations:
+  duckdb:
+    type: duckdb
+
+# your datasets are the datasets on your destinations where your data will go
+datasets: {}
+
+# your pipelines orchestrate data loading actions
+pipelines:
+  my_pipeline:
+    source: arrow
+    destination: duckdb
+    dataset_name: my_pipeline_dataset
+```
+
+:::tip
+If you do not want to start with a source, destination, and pipeline, you can simply run `dlt project init tutorial`, where `tutorial` is the name of your dlt+ Project. This will generate a project with empty sources, destinations, and pipelines.
+:::
+
+Some details about the project structure above:
+
+* The `project` section could be fully omitted in this case; it will be generated to be more explicit what the default settings are.
+* The `runtime` section is analogous to the config.toml [runtime] section and could also be omitted in this case.
+* The `profiles` section is not doing much in this case. There are two implicit profiles: `dev` and `tests` that are present in any project; we will learn about profiles in more detail later.
+
+### Substitute notation
+
+You can reference environment variables in the `dlt.yml` file using the `${ENV_VARIABLE_NAME}` syntax. Additionally, dlt+ provides several predefined project variables that are automatically substituted during loading:
+
+* `project_dir` - the root directory of the project, i.e., the directory where the `dlt.yml` file is located
+* `tmp_dir` - the directory for storing temporary files, can be configured in the project section as seen above, by default, it will be set to `${project_dir}_storage`.
+* `name` - the name of the project, can be configured in the project section as seen above
+* `default_profile` - the name of the default profile, can be configured in the project section as seen above
+* `current_profile` - the name of the current profile, this is set automatically when a profile is used
+
```

Comment:
This feels like something for the "features" page no? Don't think that's really relevant for a getting started tutorial